code/modules/mob/mob_helpers.dm

```diff
@@ -444,6 +444,8 @@ It's fairly easy to fix if dealing with single letters but not so much with comp
 	var/current_time = world.time
 	if(!M || !M.client || (M.shakecamera > current_time)|| M.stat || isEye(M) || isAI(M))
 		return
+	if(M.client && ((M.client.view != world.view) || (M.client.pixel_x != 0) || (M.client.pixel_y != 0))) //to prevent it while zooming, because zoom does not play well with this code
```

Comment:
```suggestion
	if(((M.client.view != world.view) || (M.client.pixel_x != 0) || (M.client.pixel_y != 0))) //to prevent it while zooming, because zoom does not play well with this code
```

Already checked on L445.